routes/v1.js

```diff
@@ -1353,17 +1408,24 @@ var resolveTask = async function(req, res, taskId, runId, target) {
 
   // Handle cases where the task doesn't exist
   if (!task) {
-    return res.status(404).json({
-      message: "Task not found"
-    });
+    return res.reportError(
+      "ResourceNotFound",
+      "Task {{taskId}} not found. Are you sure it was created?",
+      {
+        taskId
+      });
   }
 
   // Handle cases where the run doesn't exist
   var run = task.runs[runId];
   if (!run) {
-    return res.status(404).json({
-      message: "Run not found"
-    });
+    return res.reportError(
+      "ResourceNotFound",
+      "Run {{runId}} not found on task {{taskId}}.",
```

Comment:
btw, I love the improvement of these error messages... By all means make them more explanatory :)